skopt/space.py

```diff
@@ -5,98 +5,70 @@
 from scipy.stats.distributions import rv_discrete
 from scipy.stats.distributions import uniform
 
-from sklearn.preprocessing import LabelBinarizer
 from sklearn.utils import check_random_state
 from sklearn.utils.fixes import sp_version
 
+from skopt.transformers import CategoricalEncoder
+from skopt.transformers import HyperCube
+from skopt.transformers import Identity
+from skopt.transformers import Log10
+from skopt.transformers import Pipeline
 
 # helper class to be able to print [1, ..., 4] instead of [1, '...', 4]
 class _Ellipsis:
     def __repr__(self):
         return '...'
 
 
-class _Identity:
```

Comment:
Not sure I like moving everything to a top-level `transformers` module. Maybe create instead a `space/` submodule?